pkg/mark/link.go

```diff
@@ -205,7 +205,11 @@ func getConfluenceLink(
 	if page != nil {
 		// Confluence supports relative links to reference other pages:
 		// https://confluence.atlassian.com/doc/links-776656293.html
-		link = page.Links.Full
+		// Without /wiki prefix confluence renders broken links when page already exists.
+		link = fmt.Sprintf(
+			"/wiki%s",
```

Comment:
I don't think this is going to work with Confluence Server.

Reply:
Happy to adjust to server version, but don't have access to such setup. Open for any suggestion here to make it work in both cases 🙏

Reply:
At this point, I'd probably revert it back to absolute URL instead, so reverting https://github.com/kovetskiy/mark/pull/457